cachi2/core/models/output.py

```diff
@@ -31,25 +31,6 @@ def key(self) -> str:
         """
         return f"{self.name}:{self.version}"
 
-    @pydantic.validator("version")
```

Comment:
We will assume the proper version handling will be done at package manager specific level.

Reply:
:+1: 

maybe we should remove this after every (currently implemented) package manager gets proper handling though